testsuite/features/step_definitions/docker_steps.rb

```diff
@@ -65,6 +65,7 @@ def ck_container_imgs(count)
     build_timeout = 320
     Timeout.timeout(build_timeout) do
       step %(I navigate to images webpage)
+      step %(I wait until I do not see "There are no entries to show." text)
```

Comment:
I am not sure about why this is added here.

WARNING: I a am not saying it's wrong! :-)

Reply:
Since this other scenario is also navigating to the "Images" page and checking for something on that page, then the same applies here, we need to wait until the data is actually loaded on the table to continue executing the test.

That's why I included this here :smile: